src/FunderMaps.Infrastructure/Storage/SpacesBlobStorageService.cs

```diff
@@ -23,6 +25,9 @@ namespace FunderMaps.Infrastructure.Storage
     /// </remarks>
     internal class SpacesBlobStorageService : IBlobStorageService, IDisposable
     {
+        private static readonly byte MaxKeys = Byte.MaxValue;
```

Comment:
Initialize to a constant. The fact that `MaxValue` happens to be 255 cannot be assumed.